war/pom.xml

```diff
@@ -105,7 +105,7 @@ THE SOFTWARE.
       <groupId>org.jenkins-ci</groupId>
       <artifactId>winstone</artifactId>
       <!-- Make sure to keep jetty-maven-plugin elsewhere in this file in sync with the Jetty release in Winstone -->
-      <version>5.16</version>
+      <version>5.17</version>
```

Comment:
Could we please disable dependabot for Winstone? It's always wrong because of the comment one line above.

Reply:
Submitted #5440